src/indicators/volume_weighted_average_price.rs

```diff
@@ -0,0 +1,175 @@
+use std::fmt;
+
+use crate::{Close, Next, Volume, Reset};
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Serialize};
+
+/// Volume weight average price (VWAP).
+///
+/// In finance, volume-weighted average price (VWAP) is the ratio of the value of a security or
+/// financial asset traded to the total volume of transactions during a trading session.
+/// It is a measure of the average trading price for the period.
+///
+/// # Formula
+///
+/// ![VWAP](https://wikimedia.org/api/rest_v1/media/math/render/svg/6c0a822a0a9e58a127105e818a07061a02851685)
+///
+/// Where:
+///
+/// vwap - volume weight average price
```

Comment:
Could you pls add explanations for `Pj` and `Qj` from the formula?

Reply:
i'm good bro, thanks